python/cudf/cudf/_libxx/search.pyx

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from cudf._libxx.lib cimport *
+from cudf._libxx.column cimport Column
+from cudf._libxx.table cimport Table
+from libcpp.vector cimport vector
+cimport cudf._libxx.includes.search as cpp_search
+
+
+def search_sorted(
+    Table table, Table values, side, ascending=True, na_position="last"
+):
+    """Find indices where elements should be inserted to maintain order
+
+    Parameters
+    ----------
+    table : Table
+        Table to search in
+    values : Table
+        Table of values to search for
+    side : str {‘left’, ‘right’} optional
+        If ‘left’, the index of the first suitable location is given.
+        If ‘right’, return the last such index
+    """
+    cdef unique_ptr[column] c_result
+    cdef vector[order] c_column_order
+    cdef vector[null_order] c_null_precedence
+    cdef order c_order
+    cdef null_order c_null_order
+    cdef table_view c_table_data = table.data_view()
+    cdef table_view c_values_data = values.data_view()
+
+    # Note: We are ignoring index columns here
+    c_order = order.ASCENDING if ascending else order.DESCENDING
+    c_null_order = (
+        null_order.AFTER if na_position=="last" else null_order.BEFORE
+    )
+    c_column_order = vector[order](table._num_columns, c_order)
+    c_null_precedence = vector[null_order](table._num_columns, c_null_order)
+
+    if side == 'left':
+        with nogil:
+            c_result = (
```

Comment:
Likely need the moves here to resolve the `except +` issues you're seeing:
```suggestion
            c_result = move(
```

Reply:
Perfect - Thanks!